etc/search-headless.api.md

```diff
@@ -11,13 +11,29 @@ export interface AdditionalHttpHeaders {
     'Client-SDK'?: ClientSDKHeaderValues;
 }
 
+// Warning: (ae-forgotten-export) The symbol "Address" needs to be exported by the entry point index.d.ts

```

Comment:
looks like there are some missing exports

Reply:
addressed in another pr